### DISCUSSION TITLE
Get the Control of all parameters by software

### DISCUSSION THREAD

**Discussion by javegon** [bangcorrupt/hacktribe#213] [Q&A] [answered]
I would like to ask this aspect because you, guys, are programmers and developers, and have more knowledge than me.

I am wondering how could be posible to create an interface to control all parameters independently through a software interface or through a midi controller. I saw that chrome support midi now.

Could be possible? 

midi messages or sysex messages?

**Comment by bangcorrupt** [comment, marked as answer]
Sound parameters can be controlled via CC, or the whole pattern file can be dumped via SysEx.  I'll gradually add NRPN control of all parameters.  There is a hidden sysex message (0x41) for setting individual parameters, I haven't tested it for bugs.

**Reply by javegon** [reply]
Thanks I did a test with my midi controller, and I saw I can control volume and filters of each channel at the same time with CC. 

I would like to learn how to control other parameters via SysEx, and also with other devices like old MC 808. I hope so.

Thanks
